### DISCUSSION TITLE
Pretrain model

### DISCUSSION THREAD

**Discussion by moradza** [NVIDIA-NeMo/NeMo#6890] [Q&A]
I have two datasets, each with train, val, and test dataset. I need to pretrain model with the first dataset, and then do another round of training with the second dataset. Is there any example for this? Basically, I need to call tainer.fit(model) twice but trainer reaches max_epochs with the first dataset.

**Comment by titu1994** [comment]
You need to create a new trainer and call trainer2.fit().

**Reply by moradza** [reply]
That's what I thought. But I saw this https://github.com/Lightning-AI/lightning/issues/9636.
How can I make sure that weights are copied from the last trainer?
